apps/judicial-system/web/src/components/TagCaseState/TagCaseState.tsx

```diff
@@ -27,7 +27,7 @@ export const mapCaseStateToTagVariant = (
   state?: CaseState | null,
   caseType?: CaseType | null,
   isValidToDateInThePast?: boolean | null,
-  courtDate?: string | null,
+  arraignmentDate?: string | null,
```

Comment:
This tag is also used when courtDate is set, so renaiming it is probably meaning less unless you name it neither arraignmentDate nor courtDate.

Reply:
Ah, thanks. I've renamed this to scheduledDate, what do you think about that?

Reply:
Nice